tests/expectations/lib.rs

```diff
@@ -0,0 +1,3 @@
+#![allow(dead_code)]
+#![allow(non_camel_case_types)]
```

Comment:
Partially. I still think we want to insert this into all the bindings all the time.

Also, for some reason I still see the warnings *only some of the time* in the expectations tests, and I can't figure out why... This is definitely an improvement, however.